screen2.0/src/app/applets/gene-expression/utils.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react"
+import { Accordion, AccordionDetails, AccordionSummary, Box, Button, Stack, ToggleButton, Typography } from "@mui/material"
+import { styled } from "@mui/material/styles"
+import { RIDItemList, GeneExpEntry, GeneExpressions } from "./types"
+import { Fragment } from "react"
+import { Range2D, Point2D, linearTransform2D } from "jubilant-carnival"
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore"
+import Grid2 from "@mui/material/Unstable_Grid2/Grid2"
+
+export const ToggleButtonMean = styled(ToggleButton)(() => ({
+  "&.Mui-selected, &.Mui-selected:hover": {
+    color: "white",
+    backgroundColor: "blue",
+  },
+}))
+
+export function PlotGeneExpression(props: {
+  data: GeneExpressions
+  range: Range2D
+  dimensions: Range2D
+  RNAtype: string
+  group: string
+  scale: string
+  replicates: string
+}) {
+  const [collapse, setCollapse] = useState<{ [id: string]: { expand: boolean } }>({})
```

Comment:
Is there a reason this is an object containing a boolean value instead of just a boolean?

Reply:
don't think there is, will change to just boolean